psqlgraph/psql.py

```diff
@@ -262,17 +260,20 @@ def get_edge_count(self, session=None):
         return self.edges().count()
 
     def node_merge(self, node_id=None, node=None, acl=None,
-                   label=None, system_annotations={}, properties={},
+                   label=None, system_annotations=None, properties=None,
                    session=None, max_retries=DEFAULT_RETRIES,
                    backoff=default_backoff):
+
         with self.session_scope() as local:
             if not node and not label:
-                node = self.nodes().ids(node_id).scalar()
+                node = self.nodes().ids([node_id]).scalar()
 
             elif not node and label:
                 cls = Node.get_subclass(label)
-                node = self.nodes(cls).ids(node_id).scalar()
+                node = self.nodes(cls).ids([node_id]).scalar()
 
+            properties = properties or {}
+            system_annotations = system_annotations or {}
```

Comment:
normally this kind of default handling is at the top of the function. it's okay here I suppose...